std/node/README.md

```diff
@@ -0,0 +1,7 @@
+# Deno Node compatibility
+
+This module is meant to have a compatibility layer for the
+[nodeJS standard library](https://nodejs.org/docs/latest-v11.x/api/).
+
+**Warning** : Any function of this module do not have to be refered anywhere in
```

Comment:
I assume this is "should not" instead of "do not have to"?
std/node should be able to import utilities from other std modules, but not the other way around IMO.

Reply:
Typo: referred

Reply:
> I assume this is "should not" instead of "do not have to"?
> std/node should be able to import utilities from other std modules, but not the other way around IMO.

that's what i was meaning. Sorry for my engrish